# HG changeset patch
# User Dick Marinus <dick@mrns.nl>
# Date 1370008786 -7200
# Node ID 73bfc16fbf860f110f5141d6759a5195743454b5
# Parent  6b713ccfcda11009f82ea2de4c84d16515b3f67f
use the canonical path (for usr merge, /bin/bash -> /usr/bin/bash)

Signed-off-by: Dick Marinus <dick@mrns.nl>

diff -r 6b713ccfcda11009f82ea2de4c84d16515b3f67f -r 73bfc16fbf860f110f5141d6759a5195743454b5 policy/normalize.py
--- a/policy/normalize.py	Fri May 31 15:58:23 2013 +0200
+++ b/policy/normalize.py	Fri May 31 15:59:46 2013 +0200
@@ -761,6 +761,9 @@
                                 break
                         if not found:
                             self.warn('The interpreter path %s in %s does not exist!', interp, path)
+        if os.path.realpath(interp) != interp:
+            found = True
+            normalized = os.path.realpath(interp)
        
         if found:
                 line = m.contents['line']
